modules/node/handler/node_form_response.e

```diff
@@ -228,6 +228,12 @@ feature -- Form
 			ts.set_default_value ("Preview")
 			f.extend (ts)
 
+			if a_node /= Void and then a_node.id > 0 and then has_permission ("delete " + a_name) then
+				create ts.make ("op")
+				ts.set_default_value ("Delete")
```

Comment:
This is a general comment that apply to the other classes as well.
but ok add a TODO, or FIXME .. so that we don't forget